tests/testImageDecorrelation.py

```diff
@@ -115,19 +115,19 @@ def makeFakeImages(xim=None, yim=None, svar=0.04, tvar=0.04, psf1=3.3, psf2=2.2,
         print(np.sqrt(psf1[0]**2 - psf2[0]**2))
         print('Offset:', offset)
 
-    xim = np.arange(-256, 256, 1) if xim is None else xim
+    xim = np.arange(-128, 128, 1) if xim is None else xim
```

Comment:
Please define a variable to store the value 128, and reference that here (and line 195)